drivers/backup/portworx/portworx.go

```diff
@@ -2028,6 +2028,26 @@ func (p *portworx) GetBackupName(ctx context.Context, backupUid string, orgID st
 	return "", fmt.Errorf("backup with uid '%s' not found for org '%s'", backupUid, orgID)
 }
 
+// func GetBackupStatus return backup status and reason for a given backup name.
+func (p *portworx) GetBackupStatus(backupName string, ctx context.Context, orgID string) (api.BackupInfo_StatusInfo_Status, string, error) {
```

Comment:
Shouldn't this function be named `GetBackupStatusWithReason` ?

Reply:
Changed to GetBackupStatusWithReason